### ISSUE TITLE
drawing a blank.. clean install on test system "unexpected response code.. Authentication required."

### ISSUE THREAD

**Issue by ghost** [nexcess/magento-turpentine#111]
I would highly appreciate input on whats gone wrong here:
Please double check my simple install steps on a totaly fresh debian server
The system :
debian-6.0.7-amd64

varnished magento test machine steps

freash install deb6
add new repo
http://www.dotdeb.org/instructions/
- ( php 5.4 )

```
apt-get install nginx php5-fpm php5-common php5-cli php5 php5-curl php5-dev php5-gd php5-imagick php5-memcache php5-mysql php-pear

apt-get install mysql-server phpmyadmin subversion varnish
```

svn checkout http://svn.magentocommerce.com/source/branches/1.7
( Checked out revision 165117.)

config nginx to /site/www/public_html & use php5 socket..

make db and user for magento..
follow magento install steps
magento connecte get turpentine from  http://connect20.magentocommerce.com/community/Nexcessnet_Turpentine

flushed all caches , double checked compiler was still disabled.

so the basic shop is up and running, extension installed.
got the secrete key from /etc/varnish/secret , pasted into config 
saved config , msg returned is

```
Failed to apply the VCL to 127.0.0.1:6082: Got unexpected response code from Varnish: 107 vbdtnodrdhkgvcqmmlsvkpiorupsegaz Authentication required.

The VCL file has been saved.
The configuration has been saved.
```

what can i take from this ?  it saved the settings to the magento site root /var/default.vcl but failed to update the varnish admin interface ?

now if we go to cache management , enable the new caches.. looks fine.
clicked 'Save the configuration to a VCL file.'  , is 'ok'

click 'apply config file' repsonce is `Failed to apply the VCL to 127.0.0.1:6082: Got unexpected response code from Varnish: 107 {random string} Authentication required.`

go back to system>config>Varnish Options , enable all logging.

cat /dev/null > system.log

generate some traffic with ab,
ab -n 400 -c 10 http://dev.server.url

check the log:

```
root@debian:/site/www/public_html/var/log# tail system.log
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: wishlist_sidebar
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: right.reports.product.viewed
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: right.reports.product.compared
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: right.permanent.callout
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: right.poll
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: paypal.partner.right.logo
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: sale.reorder.sidebar
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: footer
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: before_body_end
2013-03-22T06:54:57+00:00 INFO (6): TURPENTINE: Checking ESI block candidate: core_profiler
root@debian:/site/www/public_html/var/log#
```

also worth posting some related log from syslog

```
root@debian:/var/log# tail syslog
Mar 22 07:12:46 debian varnishd[14492]: CLI telnet 127.0.0.1 39357 127.0.0.1 6082 Rd "auth" "945c63a68506d7b5411c5ad6fb4bc741cf0bcdc084a2f780b3ee610a8356ce7b"
Mar 22 07:12:46 debian varnishd[14492]: CLI telnet 127.0.0.1 39357 127.0.0.1 6082 Wr 107 wfymdpqgyrqrwughgbhtqwzilubsfwpd#012#012Authentication required.
```

so, its doing 'something', but its not actually affecting anything on the front end,
ab testing shows no change at all in reqs/sec, there is nothing in http responce headers to suggest anything from varnish.
since before we varnished magento 

bottom line..
do i need to be concerned about these  "unexpected response code.. Authentication required." messages ?
It seems to be a factor at every step, what can be changed ?

Since i get these results on my dev site, and the identical results on a totally fresh server to replicate the problem i'm hoping its just a bug that we can work out.

Alistair
LiveSite: http://iluvscotland.co.uk
DevSite: not public
Server to replicate results: VBox 8core/32GB Ram/64GB SSD/ installed debian-6.0.7-amd64-DVD1.iso
